app/services/pairing_strategies/swiss.rb

```diff
@@ -29,20 +47,17 @@ def self.get_pairings(players)
     private
 
     def assign_byes!
-      players_with_byes.each do |player|
-        round.pairings.create(
-          player1: player,
-          player2: nil,
-          score1: @bye_winner_score,
-          score2: @bye_loser_score
-        )
+      players_with_byes.each_key do |player_id|
+        @thin_pairings << PlainPairing.new(@players[player_id], @bye_winner_score, nil, @bye_loser_score)
       end
     end
 
     def paired_players
       if first_round?
-        return @paired_players ||= players_to_pair.to_a.shuffle(random:).in_groups_of(2,
-                                                                                      SwissImplementation::Bye)
+        @paired_players ||= players_to_pair.to_a.shuffle(random:).in_groups_of(2,
+                                                                               nil)
+
+        return @paired_players
```

Comment:
Why was this changed? What is the effect of replacing the bye with nil? Where is it tested?